manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.ln
+PackageVersion: 2.9.3.4
+PackageLocale: en-US
+Publisher: schinagl.priv.at

```

```suggestion
Publisher: Schinagl
```
Just wanted to confirm whether this is intentional as you wrote `Schinagl` in the other PR https://github.com/microsoft/winget-pkgs/blob/591a98ea2f24bc0f0c4c8958a008e5e038cccfe5/manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml#L7